sklearn/tests/test_multiclass.py

```diff
@@ -668,6 +668,19 @@ def test_ovo_float_y():
 def test_ecoc_exceptions():
     ecoc = OutputCodeClassifier(LinearSVC(random_state=0))
     assert_raises(ValueError, ecoc.predict, [])
+    # test that base estimator raises the right error if it
+    # does not accept sparse matrices as input
+
+    class DummyNB(MultinomialNB):
+        def fit(self, X, y):
+            X = check_array(X, accept_sparse=False)
+            return super().fit(X, y)
```

Comment:
@glemaitre not sure if this is the most appropriate place to put this dummy classifier, or is it fine if it is outside the function?